.github/workflows/ci-cd.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Python & Poetry Environment
         uses: ./.github/actions/python-environment
         with:
-          python-version: ${{ matrix.python }}
+          python-version: 3.9
```

Comment:
make other versions trouble, or is the issue that this gets executed for all versions?

Reply:
No the issue was there is not matrix define for the documentation job therefore the python version was empty